pkgs/tools/misc/cpulimit/default.nix

```diff
@@ -1,27 +1,29 @@
-{lib, stdenv, fetchurl}:
+{lib, stdenv, fetchFromGitHub}:
 
 stdenv.mkDerivation rec {
   pname = "cpulimit";
-  version = "2.7";
+  version = "0.2";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/limitcpu/${pname}-${version}.tar.gz";
-    sha256 = "sha256-HeBApPikDf6MegJf6YB1ZzRo+8P8zMvCMbx0AvYuxKA=";
+  src = fetchFromGitHub {
+    owner = "opsengine";
+    repo = "cpulimit";
+    rev = "v${version}";
+    sha256 = "1dz045yhcsw1rdamzpz4bk8mw888in7fyqk1q1b3m1yk4pd1ahkh";
   };
 
-  buildFlags = with stdenv; [ (
-    if isDarwin then "osx"
-    else if isFreeBSD then "freebsd"
-    else "cpulimit"
-  ) ];
+  patches = [ ./remove-sys-sysctl.h.patch ]
+    ++ lib.optional stdenv.isDarwin ./get-missing-basename.patch;
```

Comment:
Can we make the patch so that it works on all platforms and is always included? This prevents bit rotting and breakages when people only test on linux.